apps/web/src/components/FeedGroup/FeedGroup.utils.tsx

```diff
@@ -0,0 +1,16 @@
+import { Props as FeedItemProps } from '../FeedItem/FeedItem';
+import { ReactComponent as HomeIcon } from '../../assets/homeIcon.svg';
+import { ReactComponent as PopularIcon } from '../../assets/popularIcon.svg';
+
+export const feedItems: FeedItemProps[] = [
+  {
+    svg: <HomeIcon />,
+    label: 'Home',
+    href: '',
+  },
+  {
+    svg: <PopularIcon />,
+    label: 'Popular',
+    href: '',
+  },
+];
```

Comment:
Are we sure we want to hardcode these values? I feel like the `feedItems ` would be returned from an API call.